build.gradle

```diff
@@ -967,11 +970,9 @@ project(':iceberg-open-api') {
     testFixturesImplementation project(':iceberg-gcp')
     testFixturesImplementation project(':iceberg-azure')
     testFixturesImplementation(libs.hadoop3.common) {
-      exclude group: 'log4j'
       exclude group: 'org.slf4j'
       exclude group: 'ch.qos.reload4j'
       exclude group: 'org.apache.avro', module: 'avro'
-      exclude group: 'com.fasterxml.woodstox'
```

Comment:
Added these because of the error while brining up the runtime jar that class not found.